src/Symfony/Component/PropertyInfo/Extractor/PhpDocExtractor.php

```diff
@@ -95,6 +95,18 @@ public function getShortDescription(string $class, string $property, array $cont
             }
         }
 
+        foreach ($docBlock->getTagsByName('param') as $param) {
```

Comment:
Can you update `PhpStanExtractor` as well?

Reply:
Ok, I've looked at it and it seems it needs more work for `PhpStanExtractor` than what I've done here.
So I propose to do it in another PR.